pkgs/top-level/default.nix

```diff
@@ -19,83 +25,25 @@
                && system != "x86_64-solaris"
                && system != "x86_64-kfreebsd-gnu")
 
```

Comment:
But that, if not the `noSysDirs` change, will probably be more contentious and deserving of its own PR.